runtime/libffi/z/sysvz64.s

```diff
@@ -432,51 +480,360 @@ DARGF    DS 0H                l_DOUBLE in arg area
          LA  14,4             We reached max fprs
          B CONT               Next parameter
 
-*If we have spare gprs, pass up to 12bytes
-*in GPRs. 
-*TODO: Store left over struct to argument area
-*
+*If we have spare gprs, pass up to 24 bytes in GPRs. 
 STRCT    DS 0H
-         L   WRKREG,(2176+(((DSASZ+31)/32)*32)+24)(,4)
-         CFI WRKREG,12
-         BL STRCT2
-         B CONT
-STRCT2   DS 0H
-         LA 15,STRCTS
-         LR WRKREG,GPX
-         SLL WRKREG,2
-         L  15,0(WRKREG,15)
-         BR 15
+         LG  11,0(10,5)
+         LG 15,0(,11)          type->size
+*todo NULL check?
```

Comment:
Null check if GPR11 does not contain the valid structure?

Reply:
The idea here was an error check if the struct is invalid, but ffi_call is a void function, so there isn't much we can actually do to indicate an error at the call stage, so going to remove this todo.